src/model/object/entity/entity.h

```diff
@@ -1,27 +1,19 @@
 //
-// Created by aaron on 26/09/2019.
+// Created by aaron on 07/10/2019.
 //
 
-/* Abstract base class for all objects in the world */
```

Comment:
why was this removed?

Reply:
this should be resolved (89239037ffa3599858f2ceb4eebac4cebaf243fa)

Reply:
except the head containing the date, because we're gonna remove that anyways, right?